src/main/kotlin/frc/robot/subsystems/vision/PhotonVisionIOReal.kt

```diff
@@ -25,28 +25,32 @@ class PhotonVisionIOReal(private val camera: PhotonCamera, private val robotToCa
     }
 
     override fun updateInputs() {
-        val latestResult = camera.latestResult
+        val unreadResults = camera.allUnreadResults
 
-        if (!latestResult.hasTargets()) {
-            return
-        }
+        lateinit var estimatedPose: Optional<EstimatedRobotPose>
 
-        val estimatedPose = estimator.update(latestResult)
+        for (result in unreadResults) {
+            if (!result.hasTargets()) {
+                return
+            }
```

Comment:
Not `continue`?

Reply:
d91a9df6e3b57c0dba37d88c2302a79fbf4507af